qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageAppointmentsPage.java

```diff
@@ -14,38 +14,38 @@
 
 public class ManageAppointmentsPage extends Page {
 
-	public static final By DELETE_REQUEST = By.cssSelector(".delete-item.icon-remove");
-	private static final By CANCEL_BUTTON = By.cssSelector("#searchButtons > button");
-	private static final By SEARCH = By.cssSelector("#searchButtons > button.confirm");
+	public static final By DELETE_REQUEST_ICON = By.cssSelector(".delete-item.icon-remove");
 	private static final By APPOINTMENT = By.xpath("//table[@id='appointmentTable']/div[2]/div/div/div/div[2]/div[2]/div");
-	private static final By NEXT = By.cssSelector("#selectAppointment > button.confirm");
-	private static final By SAVE = By.xpath("//div[@id='confirmAppointment']/div[2]/button[2]");
-	private static final By BOOK_APPOINTMENT = By.cssSelector("i.icon-calendar:nth-child(1)");
-	private static final By YES_DELETE_REQUEST = By.xpath("//div[@id='confirm-cancel-appointment-request']/div[2]/button");
-	private static final By VIEW_ALL_TYPES_LINK = By.cssSelector("#viewAllAppointmentTypes > a");
+	private static final By SAVE_BUTTON = By.xpath("//div[@id='confirmAppointment']/div[2]/button[2]");
+	private static final By CANCEL_BUTTON = By.cssSelector("#searchButtons button");
+	private static final By SEARCH_BUTTON = By.cssSelector("#searchButtons button.confirm");
+	private static final By NEXT_BUTTON = By.cssSelector("#selectAppointment button.confirm");
+	private static final By BOOK_APPOINTMENT_ICON = By.cssSelector("i.icon-calendar:nth-child(1)");
+	private static final By YES_DELETE_REQUEST_BUTTON = By.xpath("//div[@id='confirm-cancel-appointment-request']/div[2]/button");
```

Comment:
What do you think of `CONFIRM_CANCEL_APPOINTMENT` instead of `YES_DELETE_REQUEST_BUTTON `

Reply:
> What do you think of `CONFIRM_CANCEL_APPOINTMENT` instead of `YES_DELETE_REQUEST_BUTTON `

Could I add button on `CONFIRM_CANCEL_APPOINTMENT_BUTTON` instead to let someone know it is a button? I am following the reviews made earlier about variable names.

Reply:
I  think even though you use  CONFIRM_CANCEL_APPOINTMENT_BUTTON  . it still remains a button. does it change.

Reply:
> I think even though you use `CONFIRM_CANCEL_APPOINTMENT_BUTTON `. it still remains a button. does it change.oK Why are you calling it `YES_DELETE_REQUEST_BUTTON`

@sherrif10 you had suggested that I name it `CONFIRM_CANCEL_APPOINTMENT ` without the word button so I suggested that I add button to make it `CONFIRM_CANCEL_APPOINTMENT_BUTTON ` for clarity. I am not objecting. I called it `YES_DELETE_REQUEST_BUTTON` because the button text is `YES`

Reply:
`CONFIRM_CANCEL_APPOINTMENT_BUTTON  `  looks much better.

Reply:
> `CONFIRM_CANCEL_APPOINTMENT_BUTTON ` looks much better

@sherrif10 Ok then let me fix this and push a commit.